lbry/wallet/ledger.py

```diff
@@ -866,7 +866,7 @@ async def resolve(self, accounts, urls, new_sdk_server=None, **kwargs):
         txos = []
         urls_copy = list(urls)
         if new_sdk_server:
-            resolve = partial(self.network.new_resolve, new_sdk_server)
+            resolve = partial(self.network.go_hub_resolve, new_sdk_server)
```

Comment:
does `new_resolve` get used by anything? should it be deleted from `Network` class?